### PR TITLE
op-guide: update docker deployment

### PR DESCRIPTION
**Pull request by iamxy**
@siddontang @andelf @thinxer @shenli 
PTAL

### CODE REVIEW

File: op-guide/docker-deployment.md [hunk 1]
```diff
@@ -7,162 +7,178 @@
...
+TiKV 和 PD 可以通过指定配置文件的方式来加载更加丰富的启动参数，用于性能调优
+
+假定配置文件在宿主机上的存放路径 `/path/to/config/pd.toml` 和 `／path/to/config/tikm.toml`
+
+那么只需要稍微调整下  Docker 的启动参数，以 TiKV 为例
 
 ```bash
-mysql -h ${host1} -P 4000 -u root -D test
+docker run -d --name tikv1 \
+...
```
Comment (siddontang) [inline op-guide/docker-deployment.md]: why ... here?

Comment (iamxy) [inline op-guide/docker-deployment.md]: Highlight the adjustment of parameters

**Comment by iamxy** [comment]
@siddontang @shenli PTAL

**Comment by siddontang** [comment]
LGTM

**Comment by iamxy** [comment]
@andelf PTAL

**Comment by iamxy** [comment]
@tennix PTAL

**Comment by andelf** [comment]
LGTM

**Comment by iamxy** [comment]
@GregoryIan PTAL

**Pull request merged**